CodeStarterCamp_Week3/Person.swift

```diff
@@ -19,4 +19,25 @@ struct Person {
     func buyCoffee(coffeeShop: CoffeeShop, coffee: Coffee) {
         print("\(name)은 가지고 있는 \(money)원으로 \(coffeeShop.name)에서 \(coffeeShop.barista.name)가 만들어주는 \(coffee)를 커피를 주문했다")
     }
+    
+    func order(_ coffee: Coffee, of coffeeShop: CoffeeShop, by name: String) {
+        if let price = coffeeShop.menu[coffee] {
+            let myMoney : Int = money
+            let coffeePrice : Int = price
+            let changeMoney = myMoney - coffeePrice
+            let absChangeMoney = abs(changeMoney)
```

Comment:
```suggestion
            let changeMoney = abs(myMoney - coffeePrice)
```

한 줄로 합쳐도 가독성에 문제가 없을 것 같습니다.

또한 `changeMoney`가 적절한 네이밍인지는 잘 모르겠습니다 🤔